lib/TPP/Conversion/ConvertLinalgToXsmm/ConvertLinalgToXsmm.cpp

```diff
@@ -929,11 +946,12 @@ struct ConvertGenericToVnniBrgemm : public OpRewritePattern<linalg::GenericOp> {
 } // namespace
 
 void mlir::tpp::populateLinalgToXsmmPatterns(RewritePatternSet &patterns) {
-  patterns.add<ConvertFillOpToUnaryZero, ConvertTransposeOpToUnaryTranspose,
-               ConvertGenericToUnaryRelu, ConvertGenericToBinaryAdd,
-               ConvertGenericToBinarySub, ConvertGenericToBrgemm,
-               ConvertBatchReduceMatmulToBatchReduceMatmul,
-               ConvertMatmulToMatmul, ConvertGenericToUnaryIdentity,
-               ConvertVnniPacking, ConvertGenericToVnniBrgemm>(
-      patterns.getContext());
+  patterns
+      .add<ConvertFillOpToUnaryZero, ConvertTransposeOpToUnaryTranspose,
```

Comment:
This is getting bloated. Can we have a single rewrite with all matchers inside? At least one for unary, one for binary and one for gemms.